compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToLLVM.cpp

```diff
@@ -67,17 +67,18 @@ void ConvertToDynamicSharedMemory(ModuleOp moduleOp) {
     }
     auto loc = addressOfOp.getLoc();
     builder.setInsertionPoint(addressOfOp);
-    LLVM::AddressOfOp globalPtr =
-        builder.create<LLVM::AddressOfOp>(loc, global);
+    LLVM::AddressOfOp globalPtr = builder.create<LLVM::AddressOfOp>(
```

Comment:
This should be kept as is to return an opaque pointer

Reply:
Ah! OK. I had started in this direction but ran into iree-opt mlir tests expecting a typed pointer. Thanks!